modules/ROOT/pages/notifications/index.adoc

```diff
@@ -93,6 +93,25 @@ Additional helper methods are exposed for some useful fields.
 !===
 |===
 
+=== Standard-defined GQLSTATUS notification codes
+
+The GQL standard defines a set of GQLSTATUS codes that are used to represent successful completion, warnings, and informational messages.
+The following table lists the standard-defined GQLSTATUS codes and their meanings:
+
+.Standard-defined GQLSTATUS notification codes
+[options="header", cols="<1s,<4"]
+|===
+|GQLSTATUS code | StatusDescription
+| 00000	        | note: successful completion
+| 00001	        | note: successful completion - omitted result
+| 01000	        | warn: warning
+| 01004	        | warn: string data, right truncation
+| 01G03	        | warn: graph does not exist
+| 01G04	        | warn: graph type does not exist
+| 01G11	        | warn: null value eliminated in set function
+| 02000	        | note: no data
+| 03000	        | info: informational
+|===
```

Comment:
I can't remember what we said about this, should we keep it or not? I'm open for either, we can always revisit later.

Reply:
I think it makes sense for notifications, as the list is short, and it looks good in the structure, but I also don't have a strong opinion. Should I create a card to change the format of the notification parameters?